packages/react-vapor/src/components/navigation/tests/NavigationConnected.spec.tsx

```diff
@@ -103,6 +103,6 @@ describe('<NavigationConnected />', () => {
 
         expect(
             _.findWhere(store.getState().paginationComposite, {id: `pagination-${basicNavigationProps.id}`}).pageNb
-        ).toBe(10);

```

Comment:
This Test was an error. If the `totalPages` is `10`. The `pageNb` can never be `10` because the `lastPage` will be `totalPage -1`. So this test was simulating a click on a page higher than the last one.